File: programs-basic-java/src/test/java/com/local/numbertypes/NeonNumberTest.java
package com.local.numbertypes;

import static com.local.numbertypes.NumberType.NEON_NUMBER;

import java.util.List;

import org.junit.jupiter.api.Test;

class NeonNumberTest extends AbstractNumberTest {

	@Test
	void testNeonNumbers() {
		testNumbersTypeByFile(NEON_NUMBER, "neonnumbers.txt");
	}

	@Test
	void testNeonNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(NEON_NUMBER, List.of(1,9), 2);
	}
}

File: programs-basic-kotlin/src/test/kotlin/com/local/numbertypes/NeonNumberTest.kt
package com.local.numbertypes
import org.junit.jupiter.api.Test
class NeonNumberTest : AbstractNumberTest() {
	@Test
	fun testNeonNumbers() {
		testNumbersTypeByFile(com.local.numbertypes.NumberType.NEON_NUMBER, "neonnumbers.txt")
	}

	@Test
	fun testNeonNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(com.local.numbertypes.NumberType.NEON_NUMBER, java.util.List.of<Int>(1, 9), 2)
	}
}